### PR TITLE
Drop support for EOL Python 2.6

### PR DESCRIPTION
**Pull request by hugovk**
Same as #1017 but only for Python 2.6.

### CODE REVIEW

File: test/test_evaluate/test_pyc.py [hunk 1]
```diff
@@ -45,20 +45,5 @@ def generate_pyc():
...
-    The list of completion must be greater than 2.
-    """
-    try:
-        generate_pyc()
-        s = jedi.Script("from dummy_package import dummy; dummy.", path='blub.py')
-        assert len(s.completions()) >= 2
-    finally:
-        shutil.rmtree("dummy_package")
-
-
```
Comment (davidhalter) [inline test/test_evaluate/test_pyc.py]: Can you not remove this test? Sorry I think the check is the wrong way around 
and this was already fixed in the virtualenv branch (will only result in merge
conflicts).

**Comment by hugovk** [comment]
Sure, will update those!

**Comment by hugovk** [comment]
Done, and rebased on master.

File: jedi/_compatibility.py [hunk 2]
```diff
@@ -242,11 +232,6 @@ def u(string):
 
 
 def literal_eval(string):
-    # py3.0, py3.1 and py32 don't support unicode literals. Support those, I
-    # don't want to write two versions of the tokenizer.
-    if is_py3 and sys.version_info.minor < 3:
-        if re.match('[uU][\'"]', string):
-            string = string[1:]
```
Comment (davidhalter) [inline jedi/_compatibility.py]: Ha! You even caught this one. Good job :)

**Comment by davidhalter** [comment]
Thanks a lot man! That really helped. Saved me quite a few hours that I can now invest in other areas to improve jedi :)

**Comment by hugovk** [comment]
Happy to help! I'll rebase https://github.com/davidhalter/jedi/pull/1019 on master later.

**Pull request merged**